vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/PackageId.java

```diff
@@ -209,14 +209,16 @@ public PackageId(String group, String name, String version) {
     public PackageId(String group, String name, Version version) {
         fromPath = false;
         // validate group
-        if (group.equals(ETC_PACKAGES)) {
+        if (group == null || group.equals(ETC_PACKAGES)) {
             group = "";
         } else if (group.startsWith(ETC_PACKAGES_PREFIX)) {
             group = group.substring(ETC_PACKAGES_PREFIX.length());
         } else if (group.startsWith("/")) {
             group = group.substring(1);
         }
         this.group = group;
+        if (name == null)
```

Comment:
this change is not related to the issue (and uses wrong coding convention)

Reply:
removing